python/ray/_private/worker.py

```diff
@@ -2708,14 +2708,14 @@ def put(
 @PublicAPI
 @client_mode_hook
 def wait(
-    ray_waitables: List[Union["ObjectRef[R]", "ObjectRefGenerator[R]"]],
+    ray_waitables: List[Union[ObjectRef, ObjectRefGenerator]],
     *,
     num_returns: int = 1,
     timeout: Optional[float] = None,
     fetch_local: bool = True,
 ) -> Tuple[
-    List[Union["ObjectRef[R]", "ObjectRefGenerator[R]"]],
-    List[Union["ObjectRef[R]", "ObjectRefGenerator[R]"]],
+    List[Union[ObjectRef, ObjectRefGenerator]],
```

Comment:
Is `R` removed so that we can generate link to ObjectRef?

Reply:
Yes, although if you'd prefer to keep `R`, we can import it at the top and instead have these be `ObjectRef[R]`. So depending on how you'd like to handle this, we can either keep it as is or add the `TypeVar`.

Reply:
@rkooo567 what do you want here since you added `R` I think